apps/extension/src/components/Request/index.tsx

```diff
@@ -0,0 +1,160 @@
+import { View, Image, Text, TouchableOpacity } from '@walless/ui';
+import { FC } from 'react';
+import styles from './style';
+import { LinearGradient } from 'expo-linear-gradient';
+import { requestType } from 'utils/config';
+interface RequestProps {
+  type: string;
+  rules?: RuleProps[];
+  buttonPrimaryContent: string;
+}
+interface RuleProps {
+  icon: string;
+  description: string;
+}
+const Request: FC<RequestProps> = ({ type, rules, buttonPrimaryContent }) => {
+  return (
+    <View className="request__container w-screen h-screen">
+      <View className="request__header h-[42px] relative">
```

Comment:
seems that you're using `css` for styling? we don't use direct `css`, just `tailwind-css` classNames or [Stylesheet.create](https://reactnative.dev/docs/stylesheet#create).